src/test/scala/nl/knaw/dans/easy/stage/fileitem/FileItemCsvSpec.scala

```diff
@@ -0,0 +1,46 @@
+package nl.knaw.dans.easy.stage.fileitem
+
+import java.io.{ByteArrayInputStream, FileInputStream}
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class FileItemCsvSpec extends FlatSpec with Matchers {
+
+  val commandLineArgs = "target/test/sdo-set".split(" ")
```

Comment:
A pattern I developed for command line arguments in general. A simple space is easier to read than a blurry set of quotes and comma's. Compare `"-f some/file -urn somestring --xyz abc".split(" ")` with `Array("-f", "some/file", "-urn", "somestring", "--xyz", "abc")` I like to treat the same things the same way everywhere.